pkg/cmd/server/kubernetes/master_config_test.go

```diff
@@ -93,11 +98,13 @@ func TestCMServerDefaults(t *testing.T) {
 			LookupCacheSizeForRC:              4096,
 			LookupCacheSizeForRS:              4096,
 			LookupCacheSizeForDaemonSet:       1024,
+			ConfigureCloudRoutes:              true,
```

Comment:
Yes, it's a new flag that is strictly more correct when the other flag is
enabled.  The other flag controls whether this flag matters, so we want
this flag on.

On Thu, Jul 7, 2016 at 3:44 PM, David Eads <notifications@github.com> wrote:

> In pkg/cmd/server/kubernetes/master_config_test.go
> <https://github.com/openshift/origin/pull/9578#discussion_r69973590>:
>
> > @@ -93,11 +98,13 @@ func TestCMServerDefaults(t *testing.T) {
> >  			LookupCacheSizeForRC:              4096,
> >  			LookupCacheSizeForRS:              4096,
> >  			LookupCacheSizeForDaemonSet:       1024,
> > +			ConfigureCloudRoutes:              true,
>
> I don't know what this does. Do you know or have you disabled it? Again,
> if not, disable it in our config and open an issue to figure out what its
> doing.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9578/files/b2902157077dc7f476bc028c0cf3ec517871a436..0d6cd9c8e4ca59d63b9d3e70ee6d32af120e7469#r69973590>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p2vh4cd30pco8_TCjrwVOSQ-hr7zks5qTVcxgaJpZM4I-xTE>
> .
>